### PR TITLE
Issue 5501b

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -158072,21 +158072,42 @@ property_value: confidence "0.9259259259259258" xsd:double
...
+xref: MESH:C566046 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
+xref: OMIM:188740 {source="Orphanet:3332", source="Orphanet:988/ntbt", source="Orphanet:988", source="Orphanet:3332/ntbt", source="MONDO:equivalentTo"}
+xref: Orphanet:3332 {source="MONDO:equivalentObsolete", source="OMIM:188740"}
+xref: Orphanet:988 {source="MONDO:equivalentTo", source="OMIM:188740"}
+xref: SCTID:716741008 {source="MONDO:equivalentTo"}
 xref: UMLS:C1861099 {source="MONDO:ncbi_mim2gene_medline", source="OMIM:188740"}
+xref: UMLS:C1861099 {source="Orphanet:988", source="MONDO:equivalentTo", source="Orphanet:988/e"}
+xref: UMLS:CN204341 {source="MONDO:equivalentObsolete"}
 is_a: MONDO:0003847 ! Mendelian disease
+is_a: MONDO:0018234 ! dysostosis
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:158102]: we should have a source for all is_a assertions

**Review by nicolevasilevsky — COMMENTED** [review]
one minor comment, feel free to merge after

**Comment by sabrinatoro** [comment]
Thank you @nicolevasilevsky. 
These "is_a" axioms came from the term merged (ie I did not create them), therefore I would have expected them to have a source. I will check with the previous release whether they didn't have a source originally or if the sources were lost during the merging process. Let's keep an eye on this in case it is a bug.

**Comment by nicolevasilevsky** [comment]
Ah, it is probably not a bug, I suspect the previous term did not have a source. Thanks for checking though. Feel free to merge when you think this is ready.

**Pull request merged**